iothub/device/src/Transport/Amqp/AmqpConnectionHolder.cs

```diff
@@ -6,6 +6,7 @@
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.Azure.Amqp;
```

Comment:
I thought only the AmqpIot* classes had direct references to the AMQP library?

Reply:
I think that would have been a good separation to have. However, that would mean that we had to translate each received amqp message into an IoT sdk type (twin/twinCollection etc) and pass that to the amqp transport layer. I updated it so that we pass an amqp message up to the point where we return to the IotHubBaseClient itself, and this neccesitated our sdk transport layer to have visibility to the Azure.Amqp types.
I thought this trade-off was worth it for the simplification, but I'm open to other ideas (i.e. we can still achieve the separation, but we'll need to keep passing multiple types around for the "receive" operation).

Reply:
I'm not sure, that's why I'm asking. @timtay-microsoft, does it matter?

Reply:
I'm not aware of any such separation that we were maintaining between the AmqpIot classes and the AMQP library. I don't even understand why we even have the separate Amqp vs AmqpIot folder structure here, though.

Reply:
It's more of a logical separation

Reply:
Tim, if we don't need to keep it and we don't have a good explanation of why there are 2 layers, we might as well consolidate them.

Reply:
I think this is something that we can make a note of and refactor later on. These layers aren't visible to the user, so we can definitely make more changes after the preview is out.